src/workerd/api/node/buffer.h

```diff
@@ -102,6 +103,14 @@ class BufferUtil final: public jsg::Object {
     // For StringDecoder
     JSG_METHOD(decode);
     JSG_METHOD(flush);
+
+    JSG_STATIC_CONSTANT_NAMED(ASCII, static_cast<EncodingValue>(Encoding::ASCII));
```

Comment:
If we find ourselves adding more enums regularly, it would be cool to add some support to JSG to just export an entire enum without all this boilerplate.

Reply:
Agreed, but it is highly unlikely. The other option is to use a JSG Object which will create a heavy JS object.